Examples/Run/ReadCsv/GenericReadCsvExample.cpp

```diff
@@ -16,72 +16,75 @@
 #include "ActsExamples/Io/Csv/CsvPlanarClusterWriter.hpp"
 #include "ActsExamples/Options/CommonOptions.hpp"
 #include "ActsExamples/Printers/PrintHits.hpp"
+#include "ActsExamples/Printers/PrintParticles.hpp"
 #include "ActsExamples/Utilities/Options.hpp"
 
 #include <memory>
 
+using namespace ActsExamples;
+
 int main(int argc, char* argv[]) {
   GenericDetector detector;
 
   // setup and parse options
-  auto desc = ActsExamples::Options::makeDefaultOptions();
-  ActsExamples::Options::addSequencerOptions(desc);
-  ActsExamples::Options::addGeometryOptions(desc);
-  ActsExamples::Options::addMaterialOptions(desc);
-  ActsExamples::Options::addInputOptions(desc);
+  auto desc = Options::makeDefaultOptions();
+  Options::addSequencerOptions(desc);
+  Options::addGeometryOptions(desc);
+  Options::addMaterialOptions(desc);
+  Options::addInputOptions(desc);
   detector.addOptions(desc);
-
-  auto vm = ActsExamples::Options::parse(desc, argc, argv);
-  if (vm.empty()) {
+  auto vars = Options::parse(desc, argc, argv);
```

Comment:
This is called `vm` in all the examples, I think.

Reply:
The naming aims to be consistent with our option typedefs, i.e. `ActsExamples::Options::Description` and `ActsExamples::Options::Variables`. If I read `vars`, I think it is more obvious what this contains. I also do not think that this must be the same everywhere.

Reply:
I don't know. This looks like "change for the sake of change" to me, but ok.

Reply:
Also gone from the diff.